lib/scrub/cip/connection_manager.ex

```diff
@@ -114,8 +114,22 @@ defmodule Scrub.CIP.ConnectionManager do
   end
 
   def encode_service(:unconnected_send, opts) do
-    request_path = opts[:request_path]
+    request_path = encode_request_path(opts[:request_path])
 
+    request_words = (byte_size(request_path) / 2) |> floor
+
+    read_elements = opts[:read_elements] || 1
+
+    <<
+      0::size(1),
```

Comment:
Subjective.

FYI in case you don't know but `0::size(1)` is not only the same as `0::1` but it is also the same as just `0`.

Reply:
I don't think this is accurate.
while  `0::size(1)` and `0::1` are the same, just `0` would be equivalent to `0::size(1)-unit(8)`. The default for bitstrings is a byte.

Reply:
Yea it was my understanding that unless specified, the 0 would be 8bits long. 
Now I have a personal preference question for you guys. I personally prefer to use ::size(#) almost allways for readability, but do you guys no of any consequences to that? Is there a reason to choose one over the other?

Reply:
@mobileoverlord ah yeah you're right, I forgot that an element of a bitstring with an implicity type defaults to an integer instead of a binary. The former's unit being bits, the latter's being bytes.

@kalbozo I prefer consistency in the codebase, so pick one and always do it in that codebase. Sounds like that is `::size(#)` for you.